src/wrappers/test-wrapper-options.ts

```diff
@@ -1,12 +1,12 @@
-import { TestLog } from "../logging/test-log";
-
-export class TestWrapperOptions {
-    name: string;
-    logger: TestLog;
-    testCases: Set<string> = new Set<string>();
-    defects: Set<string> = new Set<string>();
-
-    constructor(name: string) {
-        this.name = name;
-    }
+import { DefectPluginManager } from "../integrations/defects/defect-plugin-manager";
+import { TestCasePluginManager } from "../integrations/test-cases/test-case-plugin-manager";
+import { TestLog } from "../logging/test-log";
+
+export interface TestWrapperOptions {
+    defects?: string[];
+    defectPluginManager?: DefectPluginManager;
+    logger?: TestLog;
+    testCases?: string[];
+    testCasePluginManager?: TestCasePluginManager;
+    description?: string;
```

Comment:
need to add _BuildInfoPluginManager_ support since it is used in the _GenerateTestResult_ area of _TestWrapper_